jobs/JGLOBAL_AERO_ANALYSIS_INITIALIZE

```diff
@@ -10,8 +10,12 @@ source "${HOMEgfs}/ush/jjob_header.sh" -e "aeroanlinit" -c "base aeroanl aeroanl
 GDATE=$(date --utc +%Y%m%d%H -d "${PDY} ${cyc} - ${assim_freq} hours")
 gPDY=${GDATE:0:8}
 gcyc=${GDATE:8:2}
-GDUMP="gdas"
-
+GDUMP="gcdas"
+# Restart conditions for GFS cycle come from GDAS
+# Restart conditions for GCAFS cycle come from GCDAS
+GDUMP="${RUN}"
+GDUMP="${RUN/gfs/gdas}"
+GDUMP="${RUN/gcafs/gcdas}"
```

Comment:
What's happening here? Why is `GDUMP` set so many times and different each time?

Reply:
My understanding of this syntax, and please someone who knows more Bash than me correct me, is that this says:
```
replace gfs with gdas and replace gcafs with gcdas
```

Reply:
Ah, thanks for explaining!